app/controllers/v2/security_guides_controller.rb

```diff
@@ -20,7 +20,7 @@ def security_guides
     end
 
     def security_guide
-      @security_guide ||= authorize(resource.find(permitted_params[:id]))
+      @security_guide ||= authorize(pundit_scope.find(permitted_params[:id]))
```

Comment:
This is totally out of the context of this ticket/PR, but it has to be fixed for sure. Can you please extract this into a refactoring commit?